src/common/meta/src/error.rs

```diff
@@ -70,7 +70,7 @@ pub enum Error {
     #[snafu(display("Get null from cache, key: {}", key))]
     CacheNotGet { key: String, location: Location },
 
-    #[snafu(display("Failed to request MetaSrv, source: {}", source))]
+    #[snafu(display("{source}"))]
```

Comment:
Why remove the error message?

Reply:
I'd like to display the error message that solely belong to the embedded BoxedError. At last, it's used in an adapter.